### PR TITLE
allow multiple activities

### LINKED ISSUE #691: Streaming not recognized properly with custom status
Purple streaming on profile is not recognized when a status update happens.

In the words of Vampire: 
"The PRESENCE_UPDATE has a game field and an activities field.
The game is what is also shown as status in the client and is what we return on getActivity and what the event is fired about.
The activities is an array where your custom status and your twitch streaming is contained, but Javacord does not currently evaluate that field, I guess it is something new." -- Vampire

He also explained that javacord only evaluates game and ignores the activities

Javacord version: 3.1.2

### PR DESCRIPTION
**Pull request by Joshix-1**
fixes #691

### CODE REVIEW

File: javacord-api/src/main/java/org/javacord/api/entity/user/User.java [hunk 1]
```diff
@@ -76,11 +76,11 @@ default boolean isBotOwner() {
...
 
     /**
-     * Gets the activity of the user.
+     * Gets the activities of the user.
      *
-     * @return The activity of the user.
+     * @return The activities of the user.
      */
-    Optional<Activity> getActivity();
+    Collection<Activity> getActivities();
```
Comment (Bastian) [inline javacord-api/src/main/java/org/javacord/api/entity/user/User.java:83]: This should be a `Set<Activity>` (if the order does not matter) or a `List<Activity>` (if the order does matter) instead. 

See https://github.com/Javacord/Javacord/issues/562

File: javacord-api/src/main/java/org/javacord/api/event/user/UserChangeActivityEvent.java [hunk 2]
```diff
@@ -2,25 +2,25 @@
...
 
     /**
-     * Gets the old activity of the user.
+     * Gets the old activities of the user.
      *
-     * @return The old activity of the user.
+     * @return The old activities of the user.
      */
-    Optional<Activity> getOldActivity();
+    Collection<Activity> getOldActivities();
```
Comment (Bastian) [inline javacord-api/src/main/java/org/javacord/api/event/user/UserChangeActivityEvent.java:17]:
```suggestion
    Set<Activity> getOldActivities();
```

File: javacord-api/src/main/java/org/javacord/api/event/user/UserChangeActivityEvent.java [hunk 3]
```diff
@@ -2,25 +2,25 @@
...
 
     /**
-     * Gets the new activity of the user.
+     * Gets the new activities of the user.
      *
-     * @return The new activity of the user.
+     * @return The new activities of the user.
      */
-    Optional<Activity> getNewActivity();
+    Collection<Activity> getNewActivities();
```
Comment (Bastian) [inline javacord-api/src/main/java/org/javacord/api/event/user/UserChangeActivityEvent.java:24]:
```suggestion
    Set<Activity> getNewActivities();
```

Comment (Joshix-1) [reply javacord-api/src/main/java/org/javacord/api/event/user/UserChangeActivityEvent.java]: changed it :)

**Pull request merged**